core/jazz_t-vault/config/dev-config.json

```diff
@@ -0,0 +1,14 @@
+{
+  "T_VAULT_API": "{tvault-host-name}/vault/v2",
+  "T_VAULT_USERNAME": "{tvault-username}",
+  "T_VAULT_PASSWORD": "{tvault-password}",
+  "T_VAULT_LOGIN_API": "/auth/tvault/login",
```

Comment:
Some of  these paths are generic (environment agnostic), move them to global-config.json

Reply:
Done